src/content/docs/new-relic-control/agent-control/authentication.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: "Authentication"
+metaDescription: "Overview of the Agent Control authentication process."
+freshnessValidatedDate: never
+---
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+New Relic Fleet Control requires all requests coming from Agent Control to be signed, ensuring that the communication
+is authenticated and coming from a verified source.
+
+Agent Control must have a valid `L2 System Identity` and use it to sign all requests. L2 System Identities do not
+expire and can be used throughout the lifecycle of Agent Control.
+
+L2 System Identities are based on Public/Private Key pairs. The Public Key is uploaded to the System Identity Service,
+which then provides a ClientID. The Private Key is used to obtain an OAuth token and sign all requests sent to the
+Fleet Management service.
+
+
+<img
+  title="Agent Control Authentication L2 System Identity"
+  alt="Agent Control Authentication L2 System Identity"
+  src="/images/agent-control-authentication.webp"
+/>
+
+### Creating an L2 System Identity
+Creating an L2 System Identity requires an L1 System Identity. L1 System Identities have a TTL of 12 hours
+ and can be used to create multiple L2 System Identities as needed.
+
+
+An L1 System Identity consists of a ClientID and a ClientSecret. These credentials are generated during
+the Guided Installation process and are included in the Helm Chart Values.
+
+
+```
+global:
+  cluster: "cluster-name"
+  licenseKey: "*************************"
+agent-control-deployment:
+  identityClientId: "L1_CLIENT_ID"
+  identityClientSecret: "L1_CLIENT_SECRET"
+  config:
+    fleet_control:
+      fleet_id: "SAMPLE_FLEET_ID"
+      auth:
+        organizationId: "00000000-0000-0000-0000-000000000000"
+    subAgents:
+      ...
+```
+
+### L1 Entities Expiration
```

Comment:
I would point of that the job it leverages that only at installation time, but you can upgrade without issues with an expired key. (Is it the same for onHost?)